be/src/common/config.cpp

```diff
@@ -382,6 +382,7 @@ DEFINE_mInt32(max_single_replica_compaction_threads, "-1");
 
 DEFINE_Bool(enable_base_compaction_idle_sched, "true");
 DEFINE_mInt64(base_compaction_min_rowset_num, "5");
+DEFINE_mInt64(base_compaction_max_rowset_num, "20");
```

Comment:
We shoud limit comlumn num, it is waynum * column_num, because segment columns consumes memory too.

Reply:
According to rowset num, we shoud limit score?

Reply:
done